app/viewmodels/TradeJourneyStatusesViewModel.scala

```diff
@@ -55,27 +60,38 @@ object TradeJourneyStatusesViewModel {
         if (incomeCompletionStatus.isEmpty) IncomeNotCountedAsTurnoverController.onPageLoad(taxYear, business.businessId, NormalMode).url
         else IncomeCYAController.onPageLoad(taxYear, business.businessId).url
 
+      val expensesUrl =
+        if (expensesCompletionStatus.isEmpty)
+          OfficeSuppliesController.onPageLoad(taxYear, business.businessId, NormalMode).url    // TODO expenses categories page when built
+        else OfficeSuppliesController.onPageLoad(taxYear, business.businessId, NormalMode).url // TODO expenses CYA page when built
+
       (
         abroadUrl,
-        if (abroadCompletionStatus.getOrElse(false)) incomeUrl else "#"
+        if (abroadCompletionStatus.getOrElse(false)) incomeUrl else "#",
+        if (abroadCompletionStatus.getOrElse(false)) expensesUrl else "#"
       )
     }
 
-    val (abroadStatusString, incomeStatusString) =
+    val (abroadStatusString, incomeStatusString, expensesStatusString) =
```

Comment:
separate private method.

let's not use .get, it's similar what I mentioned in .head